src/test/java/jenkins/advancedqueue/util/PrioritySorterUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package jenkins.advancedqueue.util;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Let's use JUnit 4 assertions in the test class because we're using `org.junit.Test` from JUnit 4 to define the test and run it.  Let's replace the wildcard import with declared imports since there are so few imports.
```suggestion
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;
```

Reply:
Oh didn't see it there thanks for rectifying it